docs/status.md

```diff
@@ -35,7 +35,7 @@ one of the following tracking labels.
 | all_gather               |      no       |     yes*     |        no         |        no         |     no      |
```

Comment:
Let's downgrade `add`, `maximum` and `minimum`  in `status.md` to `Interpreter: no`?